plugins/logs/dao.go

```diff
@@ -95,20 +95,16 @@ func (es *elasticsearch) indexRecord(ctx context.Context, rec record) {
 	}
 }
 
-func (es *elasticsearch) getRawLogs(ctx context.Context, from, size, filter string, indices ...string) ([]byte, error) {
-	offset, err := strconv.Atoi(from)
+func (es *elasticsearch) getRawLogs(ctx context.Context, offset, startDate, endDate string, size int, filter string, indices ...string) ([]byte, error) {
```

Comment:
It would be good to have `map[string]string{}` as param for filtering logs. Consider in future we want to introduce some more filter like filter by index. It would become much easier.